staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1982,6 +1982,10 @@ type Probe struct {
 	// Defaults to 3. Minimum value is 1.
 	// +optional
 	FailureThreshold int32 `json:"failureThreshold,omitempty" protobuf:"varint,6,opt,name=failureThreshold"`
+	// Maximum number of failures allowed for the probe to be considered failed before having succeeded once.
+	// No default value. Minimum value is 1.
```

Comment:
Isn't the default value 0 in this case? This is larger than the minimum value already.

Also, there is no validation for the field.